components/Ui/UiCta.vue

```diff
@@ -49,35 +48,43 @@ const props = withDefaults(defineProps<Props>(), {
 
 defineEmits(["click"]);
 
+const config = useRuntimeConfig();
+
 const urlString = computed(() => {
   if (typeof props.url === "string") {
     return props.url;
   }
+
   return props.url.url;
 });
 
-const iconPerLinkType = computed(() => {
-  const url = props.url;
+// Resolving link type (Based on https://github.com/nuxt/nuxt/blob/v3.6.3/packages/nuxt/src/app/components/nuxt-link.ts#L179)
+const isExternalLogically = computed<boolean>(() => {
+  return urlString.value === "" || urlHasProtocol(urlString.value);
+});
 
-  if (props.label === "Under construction") {
-    return ErrorOutline16;
+const isExternalVisually = computed<boolean>(() => {
+  // Same domain
+  if (urlString.value.startsWith(config.public.siteUrl)) {
+    return false;
   }
-  if (isExternal(url as string)) {
+
+  return isExternalLogically.value;
+});
+
+const isFragment = computed<boolean>(() => {
+  return urlString.value.startsWith("#");
+});
+
+const iconPerLinkType = computed(() => {
+  if (isExternalVisually.value) {
     return Launch16;
-  }
-  if (isIdAnchor(url as string)) {
+  } else if (isFragment.value) {
```

Comment:
You have a `return` on the previous check, do you need an `else` here?

Reply:
Changed in https://github.com/Qiskit/qiskit.org/pull/3436/commits/b108d9ca02da6d1ee6ae7f2bb8968cd76865957f.